VerilogCodingStyle.md

```diff
@@ -1900,6 +1900,28 @@ separate combinational (`always_comb`) block. Ideally, sequential blocks should
 contain only a register instantiation, with perhaps a load enable or an
 increment.
 
+***All clock signals should be generated using blocking assignment even
+for clock dividers.***
+
+See #44 for details.
+
+&#x1f44d;
+```systemverilog {.good}
+// only for test bench code
+logic clk;
+initial begin
+  clk <= 1'b0;
+  forever #5 clk = ~clk;              // blocking assignment
+end
```

Comment:
This portion of the code isn't really relevant for this section. Since this guidance is about how to construct the divider, we don't need to see how `clk` is generated. I'd recommend removing it.

Reply:
> Edit: Or am I misunderstanding and you also want to show how to generate clock source stimulus in a testbench?

No, I don't.